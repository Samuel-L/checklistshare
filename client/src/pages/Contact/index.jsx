import React from 'react';
import PropTypes from 'prop-types';

import { withStyles } from 'material-ui/styles';
import Grid from 'material-ui/Grid';
import Typography from 'material-ui/Typography';

const styles = theme => ({
  base: {},
  title: {
    marginTop: '20px',
  },
  text: {
    marginTop: '10px',
  },
  link: {
    color: theme.palette.primary.A700,
    textDecoration: 'none',
    '&:hover': {
      color: theme.palette.primary.dark,
    },
  },
});

const Contact = props => (
  <Grid container>
    <Grid item xs={12}>
      <Typography
        className={props.classes.title}
        variant="display1"
        align="center"
        gutterBottom
      >
        Contact
      </Typography>
    </Grid>
    <Grid item xs={12}>
      <Typography className={props.classes.text}>
        Feedback and bug findings can be sent to samuell.person@gmail.com.
      </Typography>
      <Typography className={props.classes.text}>
        Source code can be found <a className={props.classes.link} href="https://github.com/Samuel-L/checklistshare">here</a>.
      </Typography>
    </Grid>
  </Grid>
);

Contact.propTypes = {
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
};

export default withStyles(styles)(Contact);
