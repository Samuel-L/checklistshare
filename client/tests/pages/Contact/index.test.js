import React from 'react';
import renderer from 'react-test-renderer';

import Contact from '../../../src/pages/Contact';

describe('pages: Contact', () => {
  const classes = { title: '', text: '', link: '' };
  const tree = renderer.create(<Contact classes={classes} />).toJSON();

  it('matches snapshot', () => {
    expect(tree).toMatchSnapshot(); 
  });
});
