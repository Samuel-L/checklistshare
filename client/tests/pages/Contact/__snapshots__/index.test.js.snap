// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages: Contact matches snapshot 1`] = `
<div
  className="MuiGrid-container-5"
>
  <div
    className="MuiGrid-item-6 MuiGrid-grid-xs-12-42"
  >
    <h1
      className="MuiTypography-root-95 MuiTypography-display1-99 MuiTypography-gutterBottom-112 MuiTypography-alignCenter-108 Contact-title-2"
    >
      Contact
    </h1>
  </div>
  <div
    className="MuiGrid-item-6 MuiGrid-grid-xs-12-42"
  >
    <p
      className="MuiTypography-root-95 MuiTypography-body1-104 Contact-text-3"
    >
      Feedback and bug findings can be sent to samuell.person@gmail.com.
    </p>
    <p
      className="MuiTypography-root-95 MuiTypography-body1-104 Contact-text-3"
    >
      Source code can be found 
      <a
        className="Contact-link-4"
        href="https://github.com/Samuel-L/checklistshare"
      >
        here
      </a>
      .
    </p>
  </div>
</div>
`;
